tutorials/btp-app-kyma-undeploy-cap-application/btp-app-kyma-undeploy-cap-application.md

```diff
@@ -1,29 +1,27 @@
 ---
-parser: v2
 author_name: Iwona Hahn
 author_profile: https://github.com/iwonahahn
+title: Undeploy Your CAP Application from Kyma
+description: This tutorial shows you how to undeploy your CAP application from Kyma.
 keywords: cap
 auto_validation: true
 time: 5
 tags: [ tutorial>beginner, software-product-function>sap-cloud-application-programming-model, programming-tool>node-js, software-product>sap-business-technology-platform, software-product>sap-btp\\, kyma-runtime, software-product>sap-fiori]
 primary_tag: software-product-function>sap-cloud-application-programming-model
 ---
 
-# Undeploy Your CAP Application from Kyma
-<!-- description --> This tutorial shows you how to undeploy your CAP application from Kyma.
-
 ## Prerequisites
  - [Assign a Role Collection to a User](btp-app-kyma-role-assignment)
 
 
-## You will learn
+## Details
+### You will learn
```

Comment:
This should be: ## You will learn. Also, please move this section above ## Prerequisites